packages/mux-video/test/player.test.js

```diff
@@ -20,6 +20,45 @@ describe('<mux-video>', () => {
     assert.equal(player.debug, false, 'debug is off');
   });
 
+  it('dispatches events properly', async function () {
+    const player = await fixture(`<mux-video
+      playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
+      muted
+    ></mux-video>`);
+
+    const eventMap = {};
+    VideoEvents.forEach((type) => {
+      eventMap[type] = false;
+      player.addEventListener(type, (e) => {
+        assert.equal(e.target, player);
+        eventMap[e.type] = true;
+      });
+    });
+
+    player.volume = 0.5;
+
+    try {
+      await player.play();
+    } catch (error) {
+      console.warn(error);
+    }
+
+    assert.deepInclude(eventMap, {
+      canplay: true,
+      durationchange: true,
+      emptied: true,
+      loadeddata: true,
+      loadedmetadata: true,
+      loadstart: true,
+      play: true,
+      playing: true,
+      timeupdate: true,
+      volumechange: true,
+      waiting: true,
+      resize: true,
+    });
+  });
```

Comment:
We can let the test runner know that we are done by returning a resolved promise like so
```suggestion
    });
    
    return Promise.resolve();
  });
```

Reply:
@gkatsev an async function always returns Promise.resolve() with the value it returns when the function body ends. in this case it would be return Promise.resolve(undefined). no need to be explicit

Reply:
oh didn't notice the test fail right away, it's unrelated to resolving the async function. random timeout issue

Reply:
Ah, that's true.
Though, it seemed to help with the timeout errors in https://github.com/muxinc/elements/commit/c71767c873afce33884815ceae16be824a0cea84

Reply:
I guess since we're trying to play, we should increase the timeout for this test at least to 5000, if not 12000 (I also noticed that I accidentally kept a really high timeout increase in https://github.com/muxinc/elements/commit/c71767c873afce33884815ceae16be824a0cea84#diff-358b4f843e79b6ab7642f6054352ab817b25c7b9a408090f5abe1b5475b494ecR420 and forgot to revert it back to 12000)

Reply:
oh actually this is because `AbortError: The play() request was interrupted by a new load request.` 
hmm, a new load request